packages/pwa-kit-extension-sdk/TUTORIAL.md

```diff
@@ -125,6 +125,54 @@ export default Home
 ```
 
 For now, you need to restart the dev server for new override files to take effect. Once the server is started, navigate to home page and you should see
+
+#### List Overridable Files
```

Comment:
If this is duplicated from the readme in the extension sdk, I would rather have it linked or referenced in somewhat. That will make it's maintainability easier. One top of that, I did do some 1-1 coding going over this tutorial with Evan and we both agreed that this file is too wordy and should have less educating material and more steps and results. So adding more content might not be ideal.